TuneBot/Modules/TrackModule.cs

```diff
@@ -349,37 +300,85 @@ public async Task GoodBot()
             return result;
         }
 
-        private async Task<string?> SearchSpotify(string details)
+        /// <summary>
+        /// Creates message components for a reply message
+        /// </summary>
+        /// <param name="spotifyLink">Spotify link</param>
+        /// <param name="youtubeLink">Youtube link</param>
+        /// <returns>A <see cref="MessageComponent"/> </returns>
+        private MessageComponent? CreateMessageComponentsForReply(string spotifyLink, string youtubeLink, string lastFmLink)
+        {
+            var component =
+                new ComponentBuilder()
+                    .WithButton(url: spotifyLink, label: "Spotify", style: ButtonStyle.Link, emote: Emote.Parse("<:spotify:1131691591623512155>"))
+                    .WithButton(url: youtubeLink, label: "YouTube", style: ButtonStyle.Link, emote: Emote.Parse("<:youtube:1131691587601182900>"))
+                    .WithButton(url: lastFmLink, label: "Last.fm", style: ButtonStyle.Link, emote: Emote.Parse("<:lastfm:1131844112472088607>"))
+                    .Build();
+
+            return component;
+        }
+
+        /// <summary>
+        /// Creates an embed for user display after using an /np or /nps command
+        /// </summary>
+        /// <param name="spotifyTrack">Spotify track object</param>
+        /// <returns>A <see cref="Embed"/></returns>
+        private async Task<Embed?> CreateEmbedAsync(FullTrack spotifyTrack)
+        {
+            var embed = new EmbedBuilder
+            {
+                Title = $"{Context.User.Username} is now listening",
```

Comment:
Should use `Context.User.GlobalName` here as this is the user's display name rather than their account name.